flink-runtime/src/test/java/org/apache/flink/runtime/externalresource/ExternalResourceUtilsTest.java

```diff
@@ -85,4 +88,20 @@ public void testConstructExternalResourceDriversFromConfig() throws Exception {
 		assertThat(externalResourceDrivers.size(), is(1));
 		assertTrue(externalResourceDrivers.get(resourceName) instanceof TestingExternalResourceDriver);
 	}
+
+	@Test
+	public void testGetExternalResourceInfoFromEnvironment() {
```

Comment:
It seems to me this test case does not cover getting info from environment.
It might be if we change `getExternalResourceInfo` to accept `Environment` as an argument.

Reply:
Same as above, I'd like to change it to `testGetExternalResourceInfo`.